### PR TITLE
View Home UI

### PR DESCRIPTION
**Pull request by hoang11549**
## Related Tickets:

-   [#38238](https://edu-redmine.sun-asterisk.vn/issues/38238)

### CODE REVIEW

**Comment by hoang11549** [comment]
ready

File: resources/views/layout/scrollBar.blade.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<aside class="left-sidebar" data-sidebarbg="skin6">
+    <!-- Sidebar scroll-->
+    <div class="scroll-sidebar">
+        <nav class="sidebar-nav">
+            <ul id="sidebarnav">
+                <li class="sidebar-item pt-2">
+                    <a class="sidebar-link waves-effect waves-dark sidebar-link" href="{{ route('home') }}"
+                        aria-expanded="false">
+                        <i class="far fa-clock" aria-hidden="true"></i>
+                        <span class="hide-menu">Dashboard</span>
```
Comment (quoctd-0607) [inline resources/views/layout/scrollBar.blade.php]: i18n, chỗ nào cần i18n thì fix luôn nhé

Comment (hoang11549) [reply resources/views/layout/scrollBar.blade.php]: dạ ok anh

**Comment by hoang11549** [comment]
ready

**Pull request merged**